app/javascript/components/purchase-list.vue

```diff
@@ -1,4 +1,5 @@
 <script setup lang="ts">
+import { computed } from 'vue';
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'computed' is defined but never used  @typescript-eslint/no-unused-vars